src/app/builds/build-details/build-details.component.ts

```diff
@@ -2,32 +2,38 @@ import { ConfigService } from './../../shared/config/config.service';
 import { AfterViewChecked, Component, OnDestroy, OnInit } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { BuildService, MainLog } from 'api-client';
-import { WharfProject } from 'src/app/models/main-project.model';
 import { BuildStatus } from '../../models/build-status';
+import { Title } from '@angular/platform-browser';
+
+const enum Tabs {
+  LOGS = 0,
+  ARTIFACTS,
+}
```

Comment:
nit: Suggest using PascalCasing on these enum values to match the convention of our other enums, such as those in `src/app/nav/nav.component.ts` and `src/app/providers/provider-type.enum.ts`